testsuites/syncgateway/functional/tests/test_db_online_offline_resync.py

```diff
@@ -489,16 +496,21 @@ def test_bucket_online_offline_resync_with_offline(params_from_base_test_setup,
         if resync_occured:
             break
 
-    time.sleep(10)
-
-    status = sg_client.bring_db_online(cluster_conf=cluster_conf, db="db")
-
-    log_info("online request issued !!!!! response status: {}".format(status))
-
-    time.sleep(5)
-    db_info = admin.get_db_info("db")
-    log_info("Status of db = {}".format(db_info["state"]))
-    assert db_info["state"] == "Online"
+    retries = 0
+    while retries < 3:
+        try:
+            status = sg_client.bring_db_online(cluster_conf=cluster_conf, db="db")
+            log_info("online request issued !!!!! response status: {}".format(status))
+            db_info = admin.get_db_info("db")
+            log_info("Status of db = {}".format(db_info["state"]))
+            assert db_info["state"] == "Online"
+            break
+        except AssertionError as error:
+            log_info("Status of db = {}".format(db_info["state"]))
+            retries = retries + 1
+            time.sleep(20)
```

Comment:
same as above comments, at least this sleep time should not beyond the original 10 second. In old case, you wait 10 seconds + 5 seconds, and most of time your test passes, this time, the test either pass immediately (chance is low), or you have to wait at lease 20 second to make the next try.

Reply:
yes please, can you please increase retries and have sleep time less?. This will help reducing the sleep time